README.md

```diff
@@ -27,6 +27,10 @@ fully selfhosted multi-user web app for externally storing Reddit items (saved,
 - updating::
 	1. cd into repo
 	2. update: `sh ./run.sh prod update`
+- automatic update and startup via systemd::
+	1. cd into repo
+	2. update expanse path in service file: `sed -i s./opt/expanse.$PWD.g expanse.service`

```

Comment:
```suggestion
	2. update expanse path in service file: `sed -i "s./opt/expanse.$PWD.g" ./expanse.service`
```
add quotes in case `$PWD` contains space(s), use relative path